### DISCUSSION TITLE
Nice style! How does one create Boxes with an Outline and Dropshadow like we see on the front page?

### DISCUSSION THREAD

**Discussion by ZiaCreatesIdeas** [executablebooks/meta#985] [General] [answered]
Hello all,
I'm new to Myst. I really like the way your front page has clickable boxes with outlines and a slight drop shadow. 
Is there a way to do this in Myst? Or a way I can do it on my own page? 
There are times when I want to document 'nodes' and this would be handy for that as well.
Thanks for all your contributions!
![Screen Shot 2023-03-29 at 6 34 48 AM](https://user-images.githubusercontent.com/106538664/228537278-37611b87-e018-4bac-a4ef-e0e8aa1c4847.png)

**Comment by chrisjsewell** [comment, marked as answer]
Heya, this is the sphinx extension used 😄 https://sphinx-design.readthedocs.io/en/furo-theme/

**Reply by ZiaCreatesIdeas** [reply]
Thank you so much for the fast reply! What a great feature set! Now if I can find a way to round the corners some, then I can design cards directly in Myst instead of having to roundtrip to Figma, export as image, reimport, position etc. Definitely a life saver!
